src/Hazelcast.Net/Clustering/ClusterConnections.cs

```diff
@@ -529,7 +529,7 @@ private async Task ConnectFirstAsync(CancellationToken cancellationToken)
                     // address is invalid or cannot be resolved via DNS... a DNS problem may
                     // be transient: better retry
                     isExceptionThrown = true;
-                    _logger.LogError(e, "Connection attempt has thrown.");
+                    _logger.LogError(e, "Connection attempt failed due to possible DNS error.");
```

Comment:
The actual exception reason will be printed via `e`, and we can not be 100% sure 't 's DNS problem, right? Then, it may be DNS or something else, hence I would rather say it may be DNS or some other problem,.